docker/vars.public.sh

```diff
@@ -5,4 +5,12 @@ export GP2GP_AMQP_BROKERS="amqp://activemq:5672"
 export GP2GP_MONGO_URI="mongodb://mongodb:27017"
 export GP2GP_MONGO_DATABASE_NAME="gp2gp"
 export GP2GP_MHS_OUTBOUND_URL="http://mock-mhs-adaptor:8081/mock-mhs-endpoint"
-export GP2GP_GPC_GET_URL="https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect"
+export GP2GP_GPC_GET_URL="http://gpcc:8090/B82617/STU3/1/gpconnect"
+export GP2GP_GPC_DOCUMENTS_FHIR_BASE="/fhir"
+
+export GPC_CONSUMER_URL="http://gpcc:8090"
+export GPC_CONSUMER_GPC_GET_URL="https://orange.testlab.nhs.uk"
+export GPC_CONSUMER_SDS_URL="http://wiremock:8080/"
+export GPC_CONSUMER_SEARCH_DOCUMENTS_PATH="/*/STU3/1/gpconnect/fhir/Patient/**"
+export GPC_CONSUMER_GPC_GET_PATIENT_PATH="/*/STU3/1/gpconnect/fhir/Patient"
```

Comment:
isn't GPC_CONSUMER_GPC_STRUCTURED_PATH missing?

Reply:
The default one is being used.